### DISCUSSION TITLE
datapoints/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#113] [Announcements]
# datapoints/

https://csmsapp.org/datapoints/

**Comment by OceannTwT** [comment]
这不比offer多多效果好？

**Reply by lezhangleonard** [reply]
1p3a是真的坑钱...以为我们赚的都是美元...

**Comment by ArkcS** [comment]
flash 我的超人

**Comment by banma12956** [comment]
求可以加上paper数量，陶瓷这些信息吗，万分感谢🙏

**Reply by banma12956** [reply]
抱歉是我没打开看    flash你是大救星

**Reply by xichenpan** [reply]
hhh btw 鼠标悬浮在行号上，会有一个zoom按钮，点击可以查看该条记录，不用左右拖动啦

**Reply by banma12956** [reply]
谢谢提示！❤️

**Reply by 0324wy** [reply]
如果再列下每个项目的学制和花费就更好了
